docs/DeveloperManuals/Project.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Project"
+sidebar_position: 5
+description: >
+  `Project` is an object that can be used for DORA to associate `changes(commits)`, with `deployments` and `issues`.
+---
+
+## Summary
+`Project` is an object that can be used for DORA to associate `changes(commits)`, with `deployments` and `issues`.
+
+It contains the following two models:
+ - `projects` describes a project object, including its name, creation and update time and other basic information
+ - `project_metrics` describes the mapping relationship enabled by a plugin, including the name of the project and plugin on both sides of the mapping, and basic information about plugin options.
+
+![image](project_table.jpg)
+
+## project
+
+|   **field**   | **type** | **length** | **description**               | **key** |
+| ------------- | -------- | ---------- | ----------------------------- | ------- |
+| `name`        | varchar  | 255        | name for project              | PK      |
+| `description` | longtext |            | description of the project    |         |
+| `created_at`  | datetime | 3          | created time of project       |         |
+| `updated_at`  | datetime | 3          | last updated time of project  |         | 
+
```

Comment:
We can add a `project_mapping` to demonstrate how different scopes are connected